src/Compute/Compute/help/Get-AzVMSqlServerExtension.md

```diff
@@ -59,22 +59,6 @@ This command gets the virtual machine named ContosoVM22 on the service Service08
 The command passes the results to the current cmdlet by using the pipeline operator.
```

Comment:
If correct, "This command gets the virtual machine named ContosoVM07 by using the Get-AzVM cmdlet".